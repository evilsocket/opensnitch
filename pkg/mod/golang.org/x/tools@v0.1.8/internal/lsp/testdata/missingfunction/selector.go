package missingfunction

func selector() {
	m := map[int]bool{}
	undefinedSelector(m[1]) //@suggestedfix("undefinedSelector", "quickfix")
}
