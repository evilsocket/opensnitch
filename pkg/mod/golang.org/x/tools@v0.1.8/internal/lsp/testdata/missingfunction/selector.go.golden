-- suggestedfix_selector_5_2 --
package missingfunction

func selector() {
	m := map[int]bool{}
	undefinedSelector(m[1]) //@suggestedfix("undefinedSelector", "quickfix")
}

func undefinedSelector(b bool) {
	panic("unimplemented")
}

